pygaggle/model/evaluate.py

```diff
@@ -213,6 +218,8 @@ def evaluate(self,
             mono_out = self.mono_reranker.rerank(example.query, example.documents)
             mono_texts.append(sorted(enumerate(mono_out), key=lambda x: x[1].score, reverse=True)[:self.mono_hits])
             scores.append(np.array([x.score for x in mono_out]))
+            if self.mono_cache_writer is not None:
```

Comment:
nit: the line below is equivalent and more concise.
```suggestion
            if self.mono_cache_writer:
```